azure-pipelines/doc-build.yml

```diff
@@ -0,0 +1,86 @@
+# Specify which branches you want to trigger for continous deployment and/or applicable for pull requests.
+# Otherwise it triggers all of them.
+trigger:
+- master
+
+pr:
+- master
+
+# Variable we can refernece later.
+variables:
+  system.debug: 'true'
+  ref.data.home: '$(Agent.BuildDirectory)/tardis-refdata'
+
+  ref.data.github.url: 'https://github.com/tardis-sn/tardis-refdata.git'
+  tardis.build.dir: $(Build.Repository.LocalPath)
+
+
+jobs:
+
+- job: 'Test'
+  pool:
+    vmImage: $[variables.vm_Image]
+
+# Basic setup of our virtual machine(vm).
+  strategy:
+    matrix:
+      linux:
+        vm_Image: 'Ubuntu-16.04'
+        conda: '/usr/share/miniconda'
```

Comment:
is this true?

Reply:
Actually specifying the pool earlier would be the basic setup of the Vm, if that what was your question was regarding, but this part is unnecessary unless we are testing multiple operating systems. So I can remove this all together.
There is indeed a miniconda in that path.